### PR TITLE
Update _emoticonresponse.siml

### PR DESCRIPTION
**Pull request by jbondguy007**
Added responses to impressed/surprised emoticons.

### CODE REVIEW

File: Brain/ArchiBoT/_emoticonresponse.siml [hunk 1]
```diff
@@ -71,5 +71,23 @@
...
+    				<Item>:O</Item>
+    				<Item>O:</Item>
+    				<Item>0_0</Item>
+    				<Item>O_O</Item>
+    			</Pattern>
+    			<Response>
+    				<Random>
+    					<Item>What's so surprising?</Item>
+    					<Item>Isn't that normal?</Item>
+    					<Item>You appear aggitated.</Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/_emoticonresponse.siml]: Uh, is that intended?

Comment (jbondguy007) [inline Brain/ArchiBoT/_emoticonresponse.siml]: @Thielak I think so? What are you referring to exactly?

Comment (Aareksio) [inline Brain/ArchiBoT/_emoticonresponse.siml]: `aggitated` => `agitated`

Comment (jbondguy007) [inline Brain/ArchiBoT/_emoticonresponse.siml]: Oh. ._.

**Pull request merged**